compliance_suite/job_runner.py

```diff
@@ -126,6 +126,24 @@ def tag_matcher(
                 return True
         return False
 
+    def version_matcher(
+            self,
+            compatible_versions: List[str]
+    ) -> bool:
+        """ Checks if the user provided spec version matches with the YAML test compatible versions.
+         Skips the test if versions are not matched.
+
+        Args:
+            compatible_versions (List[str]): The compatible versions defined for a YAML test
+
+        Returns:
+            (bool): If the user provided spec version is compatible with the YAML test, return True
```

Comment:
Better not to list the types in the docstrings as they are already specified as type hints in the method signature. Specifying this twice opens up a chance for type hints and type info in docs to go out of sync sooner or later.